packages/xs-vat-worker/test/escapeCompartment.js

```diff
@@ -0,0 +1,23 @@
+/* global issueCommand, Compartment */
+
+const encoder = new TextEncoder();
+function send(s) {
+  issueCommand(encoder.encode(s).buffer);
```

Comment:
In a future change, we should talk about changing the xsnap side to use Uint8Array instead of ArrayBuffer on both sides. ArrayBuffer is convenient since that’s the primitive used by String.fromArrayBuffer, but ArrayBuffer does not have `subarray` so it’s hard to use it without unnecessary allocation.